sunflower-land/sunflower-land
# [FEAT] Lemon Shark

# Description

- Adds Lemon Shark to Fishing
-  Removes Battlefish and Crimson Carp after Clash of Factions
- Apples Lemon Shark Boost

<img width="349" alt="lemon shark" src="https://github.com/user-attachments/assets/d3df0d82-4551-4a08-946a-0f0c17467389">

## How to Test

1. Add Lemon Fish to Inventory
2. Place on Farm
3. Grow Lemon and Check Boost

## Type of change


# How Has This Been Tested?

localhost

# Checklist:
